python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -229,6 +233,100 @@ def __startPauseClicked( self, button ) :
 	def __stopClicked( self, button ) :
 		self.getPlug().setValue( GafferScene.InteractiveRender.State.Stopped )
 
+###############################################################################
+# A widget presenting a summary of the messages in a render nodes messages plug
+###############################################################################
+
+class _MessageSummaryPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plug ) :
+
+		self.__summaryWidget = GafferUI.MessageSummaryWidget( criticalOnly = True, buttonToolTip = "Click to open the render log" )
+		GafferUI.PlugValueWidget.__init__( self, self.__summaryWidget, plug )
+
+		self.__summaryWidget.levelButtonClickedSignal().connect( Gaffer.WeakMethod( self.__levelButtonClicked ), scoped = False )
+
+		self._updateFromPlug()
+
+	def getToolTip( self ) :
+
+		return None
+
+	@GafferUI.LazyMethod( deferUntilPlaybackStops = True )
+	def _updateFromPlug( self ) :
```

Comment:
`_updateFromPlugs()` (plural)

Reply:
After a discussion offline, we decided to leave in the 'old form' to make it easier to audit single-plug widgets when we formalise an identification mechanism for this.